acvm-repo/acir/src/circuit/brillig.rs

```diff
@@ -27,3 +27,6 @@ pub enum BrilligOutputs {
 pub struct BrilligBytecode<F> {
     pub bytecode: Vec<BrilligOpcode<F>>,
 }
+
+/// Id for the function being called.
+pub type BrilligFunctionId = u32;
```

Comment:
This is better, but I still think it should be it's own struct type like we have for other Ids in the program.

Reply:
I have updated to a struct type. We use raw integers types for ACIR calls as well. I think this can be left for a separate PR though so I have made an issue here https://github.com/noir-lang/noir/issues/5700